File: CompanyAPIs/BusinessLogicLayer/DTOs/EmployeeDTOs/AddEmployeeDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessLogicLayer.DTOs.EmployeeDTOs
{
    public class AddEmployeeDTO
    {
        [Required]
        public string Name { get; set; }
        public string Address { get; set; }
        public int DepartmentId { get; set; }
    }
}

File: CompanyAngular/src/app/models/employees/addemployeedto.ts
export class Addemployeedto {
 
    name: string;
    address: string;
    departmentId:number;
    constructor(
        name: string ="",
        address: string ="",
        departmentId:number = 0,
    ) {
        this.name = name;
        this.address = address;
        this.departmentId = departmentId;
    }
}
